common/changes/office-ui-fabric-react/color-decimal_2019-06-10-21-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "ColorPicker: remove decimals from alpha value",
+      "type": "patch"
```

Comment:
shouldn't this be major? The consts were renamed

Reply:
Technically yes, but the consts were just added in a PR right before this (also in the 7 branch), so nobody should be using them yet.